.github/workflows/code-checker.yaml

```diff
@@ -109,5 +109,6 @@ jobs:
           if [[ ! -d "/var/tmp/ccache" ]];then
             mkdir -p /var/tmp/ccache
           fi
+          brew update
```

Comment:
As we have verified `"-Dzstd_SOURCE=BUNDLED"` is the key to fix the code checker on mac, so please revert this back ,thanks. https://github.com/milvus-io/milvus/runs/5531761338?check_suite_focus=true